### DISCUSSION TITLE
Library to Library mapping

### DISCUSSION THREAD

**Discussion by karan** [arabcoders/watchstate#300] [General] [answered]
Imagine a set up where I am syncing state from Plex to JF. I have 2 libraries in each: Movies and Movies4k.

Currently, if I play a movie in Plex then watchstate will mark the item in both libraries in JF as played. It would be nice to have an option to set up library <-> library sync so watching 4k only marks corresponding 4k item.

Thoughts?

**Comment by arabcoders** [comment, marked as answer]
Hi, Thank you for your suggestion it's indeed something i would like to support. But sadly, it's impossible to tell via the API which is which as we only use GUIDs and those are exactly the same for all versions of the movies. If someone has suggestion for implementation i am all ears.
